launcher/agent/agent.go

```diff
@@ -94,15 +147,21 @@ func (a *agent) Close() error {
 
 // MeasureEvent takes in a cel.Content and appends it to the CEL eventlog
 // under the attestation agent.
+// MeasureEvent measures to all Attest Roots.
 func (a *agent) MeasureEvent(event cel.Content) error {
-	a.tpmMu.Lock()
-	defer a.tpmMu.Unlock()
-	return a.cosCel.AppendEvent(a.tpm, cel.CosEventPCR, defaultCELHashAlgo, event)
+	for _, attestRoot := range a.measuredRots {
+		if err := attestRoot.Extend(event); err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
 // Attest fetches the nonce and connection ID from the Attestation Service,
 // creates an attestation message, and returns the resultant
 // principalIDTokens and Metadata Server-generated ID tokens for the instance.
+// When possible, Attest uses the technology-specific attestation root-of-trust
```

Comment:
I don't think this is accurate. If it is, we shouldn't fall back and just fail.

Reply:
Corrected